src/org/zaproxy/zap/extension/fuzz/ZapAddOn.xml

```diff
@@ -7,9 +7,7 @@
     <author>ZAP Dev Team</author>
     <url>https://github.com/zaproxy/zap-core-help/wiki/HelpAddonsFuzzConcepts</url>
     <changes>
-    <![CDATA[
-    Fix modification of file and file fuzzers in Windows.<br>
-    ]]>
+    <![CDATA[Added Export button to export results as CSV file]]>
```

Comment:
It's better to keep the CDATA tags in its own lines. It's easier to add new entries that way.
Consider adding the br tag at the end of the entry (if new entries are added this entry does not need to be changed).